src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { BrowserRouter as Router, Route } from 'react-router-dom'
+
+import LoginForm from './components/LoginForm'
+import RegisterForm from './components/RegisterForm'
+
+
+class App extends React.Component {
+
+	render() {
+		return (
+			<main className="login-section">
```

Comment:
This class name doesn't seem to be appropriate - it's not only login section that it wraps, there are other routes - e.g "register"